content/en/profiler/guide/isolate-outliers-in-monolithic-services.md

```diff
@@ -0,0 +1,94 @@
+
+{{< img src="profiler/guide-monolithic-outliers/2-outliers-monolith-cpu-top-endpoints.png" alt="" style="width:100%;" >}}
+
+
+The yellow peaks show that the `GET /store_history` endpoint has some intermittent usage corresponding to the anomalies identified above. However, the peaks might be due to differences in traffic to that endpoint. To understand if profiles can provide further insights, change the metric to `CPU - Average Time Per Call for Top Endpoints`:
+
+{{< img src="profiler/guide-monolithic-outliers/3-outliers-monolith-cpu-avg-time-per-call.png" alt="" style="width:100%;" >}}
+
+Now you can see that there is an intermittent spike in CPU utilization where _each call_ to `GET /store_history` takes on average three seconds of CPU time, rather than the increase being because of more calls.
+
+So, the CPU utilization spikes are not due to an increase in traffic but rather an increase in CPU usage per request. 
+
+## Isolate the impact of endpoints
+
+To figure out what is causing increased CPU usage each time `GET /store_history` is called, look at the profiling flame graph for this endpoint during one of the spikes. Select a time range where `GET /store_history` is showing more CPU utilization, scoping the profiling page to that time range. Then switch to the **Flame Graph** visualization to see the methods using the CPU at this time:
+
+{{< img src="profiler/guide-monolithic-outliers/4-outliers-monolith-flame-graph.png" alt="Your image description" style="width:100%;" >}}
+
+To better understand why the `GET /store_history` endpoint in particular is using more CPU, look at the table on the right-hand side, where you can see the endpoint is second from the top. Select just that row to focus the flame graph specifically on the CPU utilization caused by the `GET /store_history` endpoint. 
+
+Because you are investigating resource usage per request, also change the value in the drop-down menu at top of the table to `CPU Time per Endpoint Call`, to show the average resource usage per call to that endpoint instead of the average resource usage per minute.
+
+## Comparing flame graphs
+
+Now that the graph shows data for the right time and endpoint, you probably have enough data to understand what is causing the spike in CPU utilization. But if not, you can compare the flame graph for the spike with a time when utilization was more acceptable.
+
+To see if there are differences in which methods are using a lot of CPU time between a spike and normal usage, click **Compare** (on the far right side of the search field) and select `Previous 15 minutes`. This opens the Comparison view. 
+
+The view shows two graphs, labeled `A` and `B`, each representing a time range for CPU utilization per call for `GET /store_history`. Adjust the time selector for `A` so that it is scoped to a period with low CPU utilization per call:
```

Comment:
```suggestion
The view shows two graphs, labeled **A** and **B**, each representing a time range for CPU utilization per `GET /store_history` call. Adjust the time selector for **A** so that it is scoped to a period with low CPU utilization per call:
```